tests/sample_data/track_files/Link16/V2_GEV_16-05-2021T00-00-00.raw-SLOTS_JMSG.csv

```diff
@@ -0,0 +1,9 @@
+Xmt/Rcv,SlotTime,Set,FrameIndex,STN,RI,Packing,PG,Jx.y,MLI,ADDI,RC,TN,Lat,Long,Alt,Crs,Spd,Id,|,1,2,3,4,5,6,7,8
```

Comment:
@mew-nsc - if you switch the filename to have non-zero time (such as `04-05-06`) we can test that the times are taken as minutes after the filename hours - rather than a delta since the filename timestamp.

Reply:
Latest commit should have fixed this. Please can you take a look to make sure that the changes make sense.